src/apps/components/AppAlerts/utils.ts

```diff
@@ -0,0 +1,72 @@
+import { AppEventDeliveriesFragment, EventDeliveryStatusEnum } from "@dashboard/graphql";
+import moment from "moment";
+
+export type Webhook = NonNullable<AppEventDeliveriesFragment["webhooks"]>[0];
+
+const hasFailedAttemptsCheck = (webhook: Webhook) =>
+  webhook.failedDelivers && webhook.failedDelivers?.edges?.length > 0;
```

Comment:
```suggestion
const hasFailedAttemptsCheck = (webhook: Webhook) => webhook?.failedDelivers?.edges?.length > 0;
```

Reply:
Yes, I know it's makes more sense to do what you have proposed, but it seems like it doesn't work with Dashboard's TS version - it produces a TS error (it works with TS 5.7.2)